packages/blocks/src/page-block/edgeless/components/text/edgeless-text-editor.ts

```diff
@@ -395,66 +341,49 @@ export class EdgelessTextEditor extends WithDisposable(ShadowlessElement) {
   }
 
   override render() {
-    const { zoom, translateX, translateY } = this.edgeless.surface.viewport;
-    const { fontFamily, fontSize, textAlign, rotate, fontWeight } =
+    const { left, top, zoom, rotate, lineHeight } = this.getEditorPos();
+    const { text, fontFamily, fontSize, fontWeight, textAlign, color } =
       this.element;
-    const transformOrigin = this.getTransformOrigin(textAlign);
-    const offset = this.getTransformOffset(textAlign);
-    const paddingOffset = this.getPaddingOffset(textAlign);
+    const { hasMaxWidth, w } = this.element;
     const rect = getSelectedRect([this.element]);
-    const hasMaxWidth = this.element.hasMaxWidth;
-    const w = this.element.w;
-    const [x, y] = this.getVisualPosition(this.element);
-    const placeholder = this._renderPlaceholder();
-    const hasPlaceholder = placeholder !== nothing;
-
-    const transformOperation = [
```

Comment:
The consistency between edit and non-edit states during rotation now seems to rely on these transformation operations.

https://github.com/toeverything/blocksuite/assets/50035259/cfeec723-42a8-4456-b3f2-226eec0aa98d

Reply:
will fix this

Reply:
I was trying to simplify the code... but these transforms that too in this specific order are necessary, reverted this part of change